packages/vue3-vite/src/util/defaults.ts

```diff
@@ -38,53 +38,56 @@ export const CypressDevDependencies = {
 };
 
 export const VitestDevDependencies = {
-  '@nrwl/vite': '^15.4.2',
+  '@nrwl/vite': '^15.4.4',
   '@vue/test-utils': '^2.2.6',
   'happy-dom': '^8.1.1',
-  vitest: '^0.25.8',
+  vitest: '^0.26.3',
 };
 
 export const JestDevDependencies = {
+  '@babel/preset-env': '^7.20.2',
```

Comment:
We need babel to handle JS files in tests. I also added a `.babelrc` file. It might be annoying for users who only want to run JS, but it seems like a standard thing to do in other Nx generators. The normal approach is to include `@nrwl/web/babel`, but I couldn't get that to work.

Reply:
Because of this, I also removed the `jest.preset.ts` check since it's not needed anymore in the jest config templates.